variables.tf

```diff
@@ -32,6 +32,21 @@ variable "asp_capacity" {
 }
 
 
+// Application Insight Variables
+// UK South is unavailable for Application Insights
+variable "appinsights_location" {
+  type = "string"
+  default = "West Europe"
```

Comment:
Can't we have App Insights in the UK as well?

Reply:
Afraid not :( https://azure.microsoft.com/en-gb/status/ Just North Europe and West Europe for us.